### PR TITLE
Calculate code coverage

### PR DESCRIPTION
**Pull request by davidmreed**
This PR sets a minimum NPSP code coverage level of 86% application-wide. Current coverage is 87%; this allows for any imprecision while ensuring we do not meaningfully drop code coverage.

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: cumulusci.yml [hunk 1]
```diff
@@ -547,6 +547,7 @@ tasks:
                 - "UNABLE_TO_LOCK_ROW"
                 - "connection was cancelled here"
             retry_always: True
+            required_org_code_coverage_percent: 75
```
Comment (davidmreed) [inline cumulusci.yml:550]:
```suggestion
            required_org_code_coverage_percent: 86
```

**Review by force2b — APPROVED** [review]
Very nice. The only thing that would make it better is if the code coverage can be rendered in the PR UI somewhere. I realize that may require a whole other set of changes though.

File: cumulusci.yml [hunk 2]
```diff
@@ -550,7 +550,9 @@ tasks:
                 - "unable to obtain exclusive access to this record"
                 - "UNABLE_TO_LOCK_ROW"
                 - "connection was cancelled here"
+                - "None"
```
Comment (codexmarcel) [inline cumulusci.yml:553]: Ooof.

**Review by codexmarcel — APPROVED** [review]
`required_org_code_coverage_percent` is so satisfying.

**Pull request merged**